packages/docs/docs/api-reference/form-props.md

```diff
@@ -299,6 +298,31 @@ Dictionary of registered templates in the form. See [Custom Templates](../advanc
 
 A function can be passed to this prop in order to make modifications to the default errors resulting from JSON Schema validation. See [Validation](../usage/validation.md) for more information.
 
+## translateString
+
+Optional string translation function, if provided, allows users to change the translation of the RJSF internal strings.
+Some strings contain replaceable parameter values as indicated by `%1`, `%2`, etc.
+The number after the `%` indicates the order of the parameter.
+The ordering of parameters is important because some languages may choose to put the second parameter before the first in its translation.
```

Comment:
```suggestion
The ordering of parameters is important because some languages may choose to put the second parameter before the first in its translation. In addition to replaceable parameters, some of the strings support the use of markdown and simple html.

One can use the [documentation](https://github.com/rjsf-team/react-jsonschema-form/blob/main/packages/utils/src/enums.ts) of the `TranslatableString` enums to determine which enum values contain replaceable parameters and which support markdown and simple html.
```